xapi-xenopsd-simulator.opam

```diff
@@ -0,0 +1,35 @@
+opam-version: "1.2"
+name: "xapi-xenopsd-simulator"
+maintainer: "xen-api@lists.xen.org"
+authors: "xen-api@lists.xen.org"
+homepage: "https://github.com/xapi-project/xenopsd"
+dev-repo: "https://github.com/xapi-project/xenopsd.git"
+bug-reports: "https://github.com/xapi-project/xenopsd/issues"
+build: [[ "jbuilder" "build" "-p" name "-j" jobs ]]
+depends: [
+  "jbuilder" {build}
+  "xapi-xenopsd"
+  "bisect_ppx"
```

Comment:
I think `bisect_ppx` should not be a dependency of any of those opam files.
It's the coverage build that makes sure that it is installed, in general it is disabled and not required